hopper/templates/form_data/formdata_list.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+
+{% block title %}
+{% trans "Your Forms"%}
+{% endblock title %}
+
+{% block content %}
+<div class="row">
+  <div class="large-12 columns">
+    <table>
+      <thead>
+        <tr>
+          <th>title</th>
+          <th>url</th>
+          <th> </th>
+          <th> </th>
+        </tr>
+      </thead>
+      <tbody>
+      {% for form in formdata_list %}
+        <tr>
+          <td> {{ form.title }}</td>
+          <td><a href="{% url 'api-detail' form.id %}">{% url 'api-detail' form.id %}</a></td>
+          <td><a href="{% url 'form_data_update' form.id %}">{% trans "Edit" %} <i class="fa fa-pencil"> </i></a></td>
+          <td><a href="">{% trans "Delete" %} <i class="fa fa-trash-o"> </i></a></td>
+        </tr>
+      {% empty %}
+        <tr><td>You have no forms yet.</td></tr>
+      {% endfor %}
+      </tbody>
+    </table>
+    <div>
+      <a href="{% url 'form_data_create' %}">{% trans "Add Form" %} <i class="fa fa-plus"> </i></a>
```

Comment:
Please move the create button at the top and also add ``class="button success"`` to style it